### DISCUSSION TITLE
2011/09/28/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%94%AE%E5%87%BA%E4%B9%9D%E5%8D%83%E5%A4%9A%E5%86%8C%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#346] [Announcements]
# 2011/09/28/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%94%AE%E5%87%BA%E4%B9%9D%E5%8D%83%E5%A4%9A%E5%86%8C%E4%BA%86

我常常会去售书的网站看看读者对自己这本书的评论，最近发现京东，卓越两个网站长期缺货，也不知道为啥。于是想起来去跟编辑打听一下这本书的销售情况。胡主任告诉我，书已经卖出了九千多册了，前两次印刷的一万册很快就会售罄。今天正好是这本书发行两周年的纪念日。

https://blog.qizhen.xyz/2011/09/28/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%94%AE%E5%87%BA%E4%B9%9D%E5%8D%83%E5%A4%9A%E5%86%8C%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Evan
2011年12月5日 12:47 编辑
你的书，我也买了一本，上次去NI上海张江公司，和你们的同事谈到你，他们都很佩服你，牛人。

回复
ycui7
2011年10月1日 13:41 编辑
都已经飘洋过海到美国了。英文LabVIEW中，也没有几本有这水平。

回复
210jty
2011年09月30日 20:22 编辑
呵呵，主要还是书写的好，非常感谢，看你的书，帮我学习了labview。

回复
